web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts

```diff
@@ -44,7 +43,7 @@ const settings = {
   },
   outputDocTypes: {
     label: 'Output File Type',
-    field: 'document_type',
+    field: 'outputFileType',
     type: 'type',
     description: 'The type of document to create when -input_file_type is documents, sequencefile or delimited_text. Accepted values: mixed (documents only), xml, json, text, binary. Default: mixed for documents, xml for sequencefile, and xml for delimited_text.',
     options: [
```

Comment:
The description has to be changed. Also, I have to check with @aebadirad  regarding supported values

Reply:
@aebadirad ,
What are all the supported output types ?

Reply:
@srinathgit make sense, thanks for catching this up.

Reply:
xml, binary, text, json - the marklogic document types